config/environments/production.rb

```diff
@@ -118,4 +118,12 @@
   # config.active_record.database_resolver_context = ActiveRecord::Middleware::DatabaseSelector::Resolver::Session
 
   config.require_master_key = false
+
+  # See https://github.com/rails/rails/issues/29893
+  # This whitelists the hosts the application can trust when using `url_for` and related helpers
+  config.hosts = [
+    "beis-roda-prod.london.cloudapps.digital",
```

Comment:
Do we need these? The ENV["DOMAIN"] is set in both environments and will likely change.

Reply:
Good point, don't think so!